packages/design-system/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -40,63 +41,119 @@ export const WithDividers = {
 	render: (props: Story) => (
 		<Dropdown
 			{...props}
-			as={ButtonPrimary}
 			aria-label="Custom menu"
 			items={[
 				{
-					label: 'Community',
+					label: 'External link',
 					href: 'https://community.talend.com/s/?language=en_US',
+					target: '_blank',
+					type: 'link',
 				},
 				{
-					label: 'Support',
-					href: 'https://www.talend.com/technical-support/',
+					type: 'divider',
 				},
 				{
-					divider: true,
+					label: 'Link',
+					href: '/download',
+					type: 'link',
 				},
 				{
-					label: 'Downloads',
-					href: '/download',
+					label: 'Another link',
+					href: '/user',
+					type: 'link',
 				},
 				{
-					label: 'Account & Subscription',
-					href: '/subscription',
+					type: 'divider',
 				},
 				{
-					label: 'Profile preferences',
-					href: '/user',
+					label: 'Button',
+					onClick: () => action('logged out'),
+					type: 'button',
+				},
+			]}
+		>
+			<ButtonTertiary isDropdown onClick={() => {}}>
+				Dropdown
+			</ButtonTertiary>
+		</Dropdown>
+	),
+};
+
+export const WithTitle = {
+	render: (props: Story) => (
+		<Dropdown
+			{...props}
+			aria-label="Custom menu"
+			items={[
+				{
+					type: 'title',
+					label: 'This is a title',
 				},
 				{
-					divider: true,
+					type: 'divider',
 				},
 				{
-					label: 'Logout',
-					href: '/logout',
+					label: 'Link',
+					href: '/download',
+					type: 'link',
+				},
+				{
+					label: 'Another link',
+					href: '/user',
+					type: 'link',
 				},
 			]}
 		>
-			Dropdown
+			<ButtonSecondary isDropdown onClick={() => {}}>
+				Dropdown
+			</ButtonSecondary>
 		</Dropdown>
 	),
 };
 
+export const WithRouterLinks = () => (
+	<BrowserRouter>
+		<Dropdown
+			aria-label="Custom menu"
+			items={[
+				{
+					label: 'Link',
+					as: <RouterLink to="/destination" />,
```

Comment:
The links in dropdown are underlined on hover, is this ok ? 
(I guess the style is managed by the as component)

Reply:
Yes. It's managed by Linkable, we think links should all be that way